src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java

```diff
@@ -32,4 +32,22 @@ public GoogleSearchResultsImagesPage navigateToImagePage() {
 				.click();
 		return new GoogleSearchResultsImagesPage();
 	}
+	
+	public GoogleSearchResultsTimeSortPage navigateToTimeSortPage() {

```

Comment:
this not a navigation, this is a action - should be something like - sortBy("Past Hour")